pkg/operator/ceph/nfs/nfs.go

```diff
@@ -55,7 +55,7 @@ type daemonConfig struct {
 
 // Create the ganesha server
 func (r *ReconcileCephNFS) upCephNFS(n *cephv1.CephNFS) error {
-	nfsToSkipReconcile, err := controller.GetDaemonsToSkipReconcile(r.clusterInfo.Context, r.context, n.Namespace, config.NfsType, AppName)
+	nfsToSkipReconcile, err := controller.GetDaemonsToSkipReconcile(r.clusterInfo.Context, r.context, n.Namespace, "instance", AppName)
```

Comment:
Suggesting to create a new variable `instanceLabelKey := "instance"` and use it.  Just using the string `instance` around the code would be confusing.